scene/resources/environment.h

```diff
@@ -69,10 +69,10 @@ class Environment : public Resource {
 		TONE_MAPPER_ACES,
 	};
 
-	enum SDFGIYScale {
-		SDFGI_Y_SCALE_50_PERCENT,
-		SDFGI_Y_SCALE_75_PERCENT,
-		SDFGI_Y_SCALE_100_PERCENT,
+	enum DynamicGICascadeFormat {
+		DYNAMIC_GI_CASCADE_FORMAT_16x16x16,
+		DYNAMIC_GI_CASCADE_FORMAT_16x8x16,
+		DYNAMIC_GI_CASCADE_FORMAT_MAX,
 	};
```

Comment:
16×8×16 should be **before** 16×16×16 as it's faster (and lower quality), to match other Godot enums.

While this means the default current value of `SDFGI_Y_SCALE_75_PERCENT` (`1`) will map to `DYNAMIC_CASCADE_GI_FORMAT_16x16x16`, this isn't really an issue in practice because Godot doesn't save default values to scene/resource files. This means that if you were previously using the default value, you can be using the new default value even if it has to be `0` instead of `1`. This is the same approach followed in https://github.com/godotengine/godot/pull/75468.